README.md

```diff
@@ -1,69 +1,70 @@
+| Option                                    | Type                         | Default                  | Description                                                                                                                                        | Required |
+|-------------------------------------------|------------------------------|--------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|:--------:|
+| `network_management_always_script`        | boolean                      | `False`                  | Generate a systemd service and network management script always (overwrites networking.service if exist)                                           |     N    |
+| `network_management_default_type`         | [string](#type)              | `dhcp`                   | Default type to setup a interface or bridge                                                                                                        |     N    |
+| `network_management_default_bring_up`     | boolean                      | `True`                   | Bring interface up on start                                                                                                                        |     N    |
+| `network_management_default_dhcp_options` | [dict](#dhcp)                | `{}`                     | Additional options for dhcp interfaces                                                                                                             |     N    |
+| `network_management_default_gateway`      | string                       |                          | Add a default route via this gateway address                                                                                                       |     N    |
+| `network_management_pre_up`               | string                       | `""`                     | Commands to execute before any other action is performed                                                                                           |     N    |
+| `network_management_post_up`              | string                       | `""`                     | Commands to execute after all other network operations are performed                                                                               |     N    |
+| `network_management_nameservers`          | list of strings              | `['8.8.8.8', '8.8.4.4']` | List of all nameservers to use                                                                                                                     |     N    |
+| `network_management_clear_bridges`        | boolean                      | `False`                  | Remove all ovs bridges before recreate them, useful for renaming of bridges                                                                        |     N    |
```

Comment:
"Remove all ovs bridges before recreating them. This is useful for renaming bridges."